lib/plugins/aws/info/getStackInfo.js

```diff
@@ -52,6 +43,11 @@ module.exports = {
           .then(result => {
             if (result) stackData.externalHttpApiEndpoint = result.ApiEndpoint;
```

Comment:
Wonder now why we have `if (result)` here, maybe to make some stubbed tests not crash. Ideally it should be removed, as it just confuses